presto-memory/src/main/java/io/prestosql/plugin/memory/MemoryMetadata.java

```diff
@@ -211,12 +211,10 @@ public synchronized MemoryOutputTableHandle beginCreateTable(ConnectorSession se
     {
```

Comment:
typo in commit message: `refacotoring`. I would just write:
```
Refactor memory connector

- Remove redundant variable usage
- Prefer assertEquals if possible
- Prefer assertNull if possible
- Avoid raw type usage
- Remove deprecated beginInsert usage
```

Reply:
Updated the commit message. 👍